### PR TITLE
Jss

### PR DESCRIPTION
**Pull request by ItayGershman**

### CODE REVIEW

File: 3-JSS/utils/localstorage.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+export class LocalStorage {
+  constructor() {
+    if (!localStorage.getItem("todoList"))
```
Comment (tombenezra) [inline 3-JSS/utils/localstorage.js]: it's a bit weird to have such side effect on the `constructor` and not on-demand. for example when you're attempting to save todos for the first time

File: 3-JSS/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const CopyPlugin = require("copy-webpack-plugin");
+
+module.exports = {
+  mode: "development",
```
Comment (tombenezra) [inline 3-JSS/webpack.config.js]: usually you would go with a `production` mode by default for bundling and `development` settings when working with the dev-server. can you tell the difference?

File: 3-JSS/js-styles/style.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
+import jss from "jss";
+import preset from "jss-preset-default";
+
+jss.setup(preset());
```
Comment (tombenezra) [inline 3-JSS/js-styles/style.js]: I suggest separating the setup and style definitions into different files

File: 3-JSS/main.js [hunk 4]
```diff
@@ -0,0 +1,182 @@
+import { v4 as uuidv4 } from "uuid";
```
Comment (tombenezra) [inline 3-JSS/main.js:1]: has this file changed besides the way classes are imported? cause i'm not reviewing it 😅

**Review by tombenezra — CHANGES_REQUESTED** [review]
Looks very good! left a few comments

File: 3-JSS/main.js (hunk 4 above)
Comment (ItayGershman) [reply 3-JSS/main.js:1]: lol, I just change how the classes are imported 😅

File: 3-JSS/js-styles/style.js (hunk 3 above)
Comment (ItayGershman) [reply 3-JSS/js-styles/style.js]: Do you mean to make a file only for the imports and the setup and then import this file into the styling file?

File: 3-JSS/webpack.config.js (hunk 2 above)
Comment (ItayGershman) [reply 3-JSS/webpack.config.js]: Got it.
From now on the webpack.config.js mode will be "production" in order to optimize the application.

File: 3-JSS/utils/localstorage.js (hunk 1 above)
Comment (ItayGershman) [reply 3-JSS/utils/localstorage.js]: Alright, I understand. So now I am checking if the list exists only on save function

**Review by tombenezra — APPROVED** [review]

**Pull request merged**